qucs/qucs/dialogs/simmessage.cpp

```diff
@@ -594,9 +594,6 @@ void SimMessage::startSimulator()
   QString sep(":");
 #endif
 
-  // append process PATH
-  // insert Qucs bin dir, so ASCO can find qucsator
-  env.insert("PATH", env.value("PATH") + sep + QucsSettings.BinDir );
   SimProcess.setProcessEnvironment(env);
```

Comment:
now `asco` cannot find `qucsator` anymore. What should be the Right Thing? I have several `qucs` versions installed, none is in the path and of course I need `asco` to run the `qucsator` associated with the Qucs version from which is started.